lib/resources/r4/appointment_patch.yaml

```diff
@@ -19,3 +19,75 @@ operations:
       <ul>
         <li>Only "arrived", "checked-in", and "cancelled" are supported</li>
       </ul>
+  - name: add-provider-vmr
+    path: /contained/0/telecom/0/value
+    operation: add
+    cardinality: 0..1
+    type: string
+    url: todo
+    description: The provider video visit link
+    example: |
+      {
+        "op": "add",
+        "path": "/contained/0/telecom/0/value",
+        "value":  "http://providerlink.vmr.net"
+      }
+    note: >
+      <ul>
+        <li>This patch operation is for a video visit appointment</li>
+        <li>This operation must be accompanied by all other video visit patch operations</li>
```

Comment:
This patch operation must be accompanied by all other video visit link patch operations

Reply:
https://github.com/cerner/fhir.cerner.com/pull/294/commits/2662d3989c06b836a807167951a85a63068e21b5#diff-07dbf4d3264a1e7a98f8d854b77f88e9R38